guby_backend/templates/app/meetup_add_owner.html

```diff
@@ -0,0 +1,217 @@
+            color: #777;
+        }
+
+        .btn-google {
+            color: white;
+            background-color: #ea4335;
+        }
+
+        .btn-facebook {
+            color: white;
+            background-color: #3b5998;
+        }
+
+        /* Fallback for Edge
+-------------------------------------------------- */
+
+        @supports (-ms-ime-align: auto) {
+            .form-label-group>label {
+                display: none;
+            }
+
+            .form-label-group input::-ms-input-placeholder {
+                color: #777;
+            }
+        }
+
+        /* Fallback for IE
+-------------------------------------------------- */
+
+        @media all and (-ms-high-contrast: none),
+        (-ms-high-contrast: active) {
+            .form-label-group>label {
+                display: none;
+            }
+
+            .form-label-group input:-ms-input-placeholder {
+                color: #777;
+            }
+        }
+    </style>
+
+    <title>ChennaiPy | guby.live</title>
+</head>
+
+<body>
+    <!-- Navigation -->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark static-top">
+        <div class="container">
+            <a class="navbar-brand" href="#">
+                <img src="http://placehold.it/150x50?text=Logo" alt="">
+            </a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive"
+                aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarResponsive">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">Home</a>
+                    </li>
+                    <!-- <li class="nav-item active">
+                        <a class="nav-link" href="#">Sign In
+                            <span class="sr-only">(current)</span>
+                        </a>
+                    </li> -->
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-6 mx-auto">
+                <div class="card" style="margin-top:5rem;">
+                    <!-- <img class="card-img-top" src="..." alt="Card image cap"> -->
+                    <div class="card-body">
+                        <h5 class="card-title">Add Speakers</h5>
+                        <!-- <p class="card-text">Enter details of your super awesome meetup</p> -->
+
+                        {% load crispy_forms_tags %}
+                        <form class="form-signin" action="/app/meetups/{{ meetup_id }}/owner/add" method="POST">
+                            {% csrf_token %}
+
+                                <input type="text" id="inputMeetupOwner" class="form-control" placeholder="Add meetup owner"
```

Comment:
@shrayasr  - Form might require a model? I don't know how to handle a form without a model, hence this. Please point me to the right examples that does form handling without a form. (because owner is not part of any model)

Reply:
I will check on this